README.md

```diff
@@ -44,7 +44,7 @@ claims no expertise beyond English.
 | ja     | Japanese             | 4.0.3   | [kenjis][ja]         | Kenji Suzuki
 | ko     | Korean               | 4.0.2   | [codeigniter-kr][kr] | codeigniter-kr
 | lt     | Lithuanian           | 4.0.2   | [dgvirtual][lt]      | Donatas Glodenis
-| lv     | Latvian              | 4.0.3   | [davislasis][lv]     | Dāvis Lasis
+| lv     | Latvian              | 4.1.1   | [davislasis][lv]     | Dāvis Lasis
```

Comment:
Please revert this change. LV is introduced in 4.0.3

Reply:
ok, i thought it is "in what current state this language translation is in".
I must delay this commit / fix because of problems getting php-cs-fixer in windows machine. I hope in next week or so i will get a new linux server up and running and then i can re-commit.

Thanks